irodsConnector/session.py

```diff
@@ -155,12 +187,11 @@ def zone(self) -> str:
             Zone.
 
         """
-        if self.session:
-            return self.session.zone
-        return ''
+        if self._irods_session:
+            return self._irods_session.zone
+        return None
```

Comment:
An `iRODSSession` returns strings for all these attributes (they are actually `@property`s of that class) except `server_version`, which returns a tuple.  Returning `None` when the session not available is an inconsistent return value and will require _any and all_ code using these properties to check first if they exist to avoid an `AttributeError`.